frontend/src/components/filters/Filters.js

```diff
@@ -678,37 +206,43 @@ class Filters extends PureComponent {
   }
 }
 
-/**
```

Comment:
Why remove the proptypes comments ?

Reply:
I believe those are kind of redundant since Proptypes itself is pretty much self-explanatory for every field since you have the type. If you have 20 params there will be extra 20 lines. If for some special param like an object you want to add details you can put comment but if you just put @param {object} someObject then I believe is redundant.. since someObject: Proptype.object, is obvious what it is.